src/compiler/checker.ts

```diff
@@ -5529,45 +5587,45 @@ module ts {
                 if (memberDecl.kind === SyntaxKind.PropertyAssignment ||
                     memberDecl.kind === SyntaxKind.ShorthandPropertyAssignment ||
                     isObjectLiteralMethod(memberDecl)) {
-                    if (memberDecl.kind === SyntaxKind.PropertyAssignment) {
+                        if (memberDecl.kind === SyntaxKind.PropertyAssignment) {

```

Comment:
Rather than shifting everything over, I'd prefer if you just changed the condition to

```TypeScript
if (memberDecl.kind === SyntaxKind.PropertyAssignment
    || memberDecl.kind === SyntaxKind.ShorthandPropertyAssignment
    || isObjectLiteralMethod(memberDecl)) {
```